bindings/node.js/Makefile

```diff
@@ -1,40 +1,78 @@
-all: clean build format test bundle
+# For a less verbose yarn.
+YARN = yarn --silent
 
+.PHONY: all
+all: format build test bundle
```

Comment:
Random question, and also something I do not feel strongly about as a note.  @jtraglia, why not run `install` here instead of in the `build` and `test` commands?  If it is in `all`, running `make` runs through the whole process once for ease of high-level/CI review (run one command and it all "just works") and then does not run the install again on iterative builds and tests during active development to speed up the process (`make test` running on a watch process that monitors file changes)

Reply:
So `install` is a dependency of anything that requires `yarn`. Running `test` will run `install` first. It should "just work" as describe. See here, `install` is listed after `test`.

https://github.com/ethereum/c-kzg-4844/blob/c7b5e32c56524ddcfba106225f2cd098bb4c5e6e/bindings/node.js/Makefile#L55-L56

As for not including `install` in `all`, it will be called anyway so it's just a little simpler without.